InfiniteRecharge/src/main/java/frc/robot/subsystem/vision/VisionSubsystem.java

```diff
@@ -40,6 +40,8 @@ public void initialize() {
         tableInstance.startClientTeam(4183);
 
         limelightTable = tableInstance.getTable("limelight");
+
+        turnOnLEDs();
```

Comment:
WE should turn on LEDs in autoInit and teleopInit and turn them off in disableInit.